lib/api/WebApi.js

```diff
@@ -181,6 +194,14 @@ async function getReportWeb(type, id, reportParameters) {
   };
 }
 
+async function getStorage(namespace, key) {
+  const options = {
+    method: 'GET',
+    responseType: 'blob',
```

Comment:
it would be really nice to make these client-side API wrappers streaming, but we can deal with that later.